src/pages/Profile/Follower/index.tsx

```diff
@@ -0,0 +1,172 @@
+    }
+  };
+
+  const renderContent = () => {
+    if (followerList.length) return renderFollowerCards();
+    return <EmptyText>{emptyText}</EmptyText>;
+  };
+
+  const renderFollowerCards = () => {
+    return (
+      <InfiniteScroll dataLength={followerList.length} hasMore={hasMore} heightMargin={200} next={fetchMoreFollowers}>
+        {followerList.map((item, index) => {
+          const user = extractObject(item);
+          return getFollowerCard(index, item.id, item.self_following, user);
+        })}
+      </InfiniteScroll>
+    );
+  };
+
+  const updateUserStats = async () => {
+    if (self.id) await dispatch(getUserStats(self.id));
+  };
+
+  const handleFollowerFollowBtnClick = async (userID: number) => {
+    await dispatch(createFollower({following: userID}));
+    updateUserStats();
+  };
+
+  const handleFollowerUnFollowBtnClick = async (followingUserID: number) => {
+    dispatch(deleteFollower(followingUserID));
+    updateUserStats();
+  };
+
+  const handleFollowingUnFollowBtnClick = async (followerId: number, followingUserID: number) => {
+    dispatch(deleteFollowing(followerId, followingUserID));
+    updateUserStats();
+  };
+
+  const renderFollowButton = (followerId: number, user: UserReadSerializer, selfFollowing: boolean) => {
+    if (self.id !== userId) return null;
+
+    return type === FollowerType.FOLLOWERS ? (
+      <S.BtnContainer>
+        {selfFollowing ? (
+          <S.UnFollowButton onClick={() => handleFollowerUnFollowBtnClick(user.id)}>
+            <Icon path={mdiAccountMinusOutline} size={1} />
+            <S.BtnText>Unfollow</S.BtnText>
+          </S.UnFollowButton>
+        ) : (
+          <S.FollowButton onClick={() => handleFollowerFollowBtnClick(user.id)}>
+            <Icon path={mdiAccountPlusOutline} size={1} />
+            <S.BtnText>Follow</S.BtnText>
+          </S.FollowButton>
+        )}
+      </S.BtnContainer>
+    ) : (
+      <S.BtnContainer>
+        <S.UnFollowButton onClick={() => handleFollowingUnFollowBtnClick(followerId, user.id)}>
+          <Icon path={mdiAccountMinusOutline} size={1} />
+          <S.BtnText>Unfollow</S.BtnText>
+        </S.UnFollowButton>
+      </S.BtnContainer>
+    );
+  };
+
+  const getFollowerCard = (index: number, followerId: number, selfFollowing: boolean, user: UserReadSerializer) => {
+    return (
+      <S.FollowerContainer key={index}>
+        <S.Grid>
+          <S.Counter>{index + 1}</S.Counter>
+          <S.FollowerLink to={`/profile/${user.id}`}>
+            <Avatar src={user.avatar} />
+            <S.Username>{user.username}</S.Username>
+          </S.FollowerLink>
+          {renderFollowButton(followerId, user, selfFollowing)}
+        </S.Grid>
+      </S.FollowerContainer>
+    );
+  };
+
+  return (
+    <S.Container className={className}>
+      <S.Header>
+        <S.Heading>
+          {title} — {count}
+        </S.Heading>
+        {/* 
+          TODO(muhammad) MEDIUM: Implement search functionality
```

Comment:
@buckyroberts Since this PR got a little too big so can we do the search in a separate issue/PR?
I can open one